community-tutorials/pufferpanel/01-en.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Pufferpanel on a Debian/Ubuntu Server
+description: How to install the Pufferpanel on Debian or Ubuntu servers
+updated_at: 2021-11-09
+slug: install-pufferpanel-debian-ubuntu
+author_name: Jonas Löchner
+author_url: https://sancraft.dev/
+author_image: https://dl.sancraft.dev/images/Sanoj/1-B.png
+author_bio: -
+tags: [mc, minecraft, web, pufferpanel, panel, linux, debian, ubuntu]
+netcup_product_url: https://netcup.de/bestellen/produkt.php?produkt=2623
+language: en
+available_languages: en
+---
+
+# Introduction
+This tutorial will show you, how you can install the Pufferpanel on a Debian or Ubuntu Server on the server "RS 2000" trough apt and how you can set up a apache2 reverse proxy to it <br>
+The Pufferpanel is a simple Webpanel for game servers and many more! <br>
+I am doing this on a debian 11 server, but it also should work on other debian versions and on ubuntu <br>
+This Tutorial is based on the Pufferpanel Docs https://docs.pufferpanel.com/en/2.x/installing.html, on https://github.com/SanCraftDev/Debian-Setup/blob/main/README.md#Snapd and on https://github.com/SanCraftDev/Debian-Setup/blob/main/README.md#Apache-and-Certbot
+
+# Requirements
+You need a Debian or Ubuntu Server with an Internet Connection, which is accessible from the internet over a static IP on port 8080 and 5657. <br>
```

Comment:
Do I just need to open my ports via ufw?

Reply:
if you use ufw, yes